src/covid_model_seiir_pipeline/core/versioner.py

```diff
@@ -336,17 +339,27 @@ class RegressionVersion(Version):
     n_draws: int
     location_set_version_id: int
 
-    # Spline Arguments
+    # Spline required arguments
     degree: int
     knots: np.array
 
     # Regression Arguments
     covariates: Dict[str, Dict[str, Union[bool, List, float]]]
     covariates_order: List[List[str]] = None
     covariate_draw_dict: Dict[str, bool] = None
```

Comment:
The only new argument here is `sequential`. I also reordered some of the other RegressionVersion arguments into logical groups.